include/swift/Option/Options.td

```diff
@@ -454,6 +454,17 @@ def emit_module_interface_path :
          ArgumentIsPath, SupplementaryOutput]>,
   MetaVarName<"<path>">, HelpText<"Output module interface file to <path>">;
 
+def emit_module_summary :
```

Comment:
Nit: I would put these new entries either above `emit_module_interface` or below `emit_private_module_interface_path` to avoid splitting the module interface options from each other.